### PR TITLE
558 use crunchbase v4

### PR DESCRIPTION
**Pull request by AndreyKozlov1984**
This migrates API crunchbase API from V3 to V4
To ensure that data is valid I fetch data twice - first via V3 api and then via V4 api and then check that data is identical. 
Netlify builds will be run with level=medium to force a full crunchbase check, thus a build will be slower than usually and this PR should not be merged before changing package.json to the original version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "copy-iframe-resizer": "((cp \"$PROJECT_PATH\"/node_modules/iframe-resizer/js/iframeResizer.min.js \"$PROJECT_PATH\"/dist/iframeResizer.js 2>/dev/null) || (cp node_modules/iframe-resizer/js/iframeResizer.min.js \"$PROJECT_PATH\"/dist/iframeResizer.js)) && echo \"`cat \"$PROJECT_PATH\"/dist/iframeResizer.js`\n`cat src/iframeResizer.js`\" > \"$PROJECT_PATH\"/dist/iframeResizer.js",
     "setup-robots": "babel-node tools/sitemap && babel-node tools/addRobots",
     "quick-build": "babel-node tools/build.js && npm run copy-dist && npm run setup-robots",
-    "prebuild": "npm config set scripts-prepend-node-path true && npm run fetch && npm run clean-dist",
+    "prebuild": "npm config set scripts-prepend-node-path true && LEVEL=medium npm run fetch && npm run clean-dist",
```
Comment (AndreyKozlov1984) [inline package.json:39]: Don't merge

**Comment by AndreyKozlov1984** [comment]
I've reviewed logs, it seems like data should be identical.
Do not merge, I need to disable fetching crunchbase entries every time.
@dankohn do you think it is better to remove V3 api completely from our codebase, because data is identical on all landscapes ?

**Comment by dankohn** [comment]
Yes, please remove v3 entirely.

**Comment by AndreyKozlov1984** [comment]
Ok, I'll be using CRUNCHBASE_KEY_4 for new api to avoid conflicts between old and new keys.

**Comment by AndreyKozlov1984** [comment]
LGTM

**Comment by AndreyKozlov1984** [comment]
@dankohn please merge

**Pull request merged**